docs/hugo/content/guide/crd-management.md

```diff
@@ -0,0 +1,140 @@
+---
+title: CRD management in ASO
+weight: -3
+---
+
+## How does CRD management work in Azure Service Operator?
+
+ASO performs CRD management differently than many traditional operators. You might notice if you install the
+[Helm chart](../installing-from-helm) or the [YAML](../installing-from-yaml) that there are NO CRDs installed by default.
+
+This is for two reasons:
+
+1. ASO supports so many CRDs that it exceeds the maximum Helm chart size when CRDs are included, so the CRDs cannot be 
+   included in the Helm chart.
+2. ASO supports so many CRDs that it can cause problems if they're all installed onto a Kubernetes cluster with a low
+   memory limit. See [#2920](https://github.com/Azure/azure-service-operator/issues/2920) for details.
+
+For the above reasons, by default each version of the ASO operator pod embeds the CRDs inside the pod and installs them when
+the pod launches. 
+
+**No CRDs are installed by default.** New CRDs will only be installed if they are specified in the 
+`--crd-pattern` cmdline argument to the `manager` container. `--crd-pattern` is a semicolon delimited string of CRD
+patterns to apply. Each pattern is a case-insensitive glob (supports `*`, `?` globbing).
+
+**Existing CRDs are always upgraded.** If a CRD already exists in the cluster (from a previous version of ASO or 
+a manual CRD installation), it will always be updated to the latest version of the CRD. Note that upgrading a CRD doesn't
+upgrade existing resources, it just gives access to new versions of that CRD. For example `resources.azure.com/ResourceGroup`
+might be upgraded to support a new API version `v1api20230601`.
+
+## Helm
+
+### Automatic CRD installation (recommended)
+
+Use the `crdPattern` value in `values.yaml` to specify what new CRDs to install. Existing CRDs will always be upgraded
+even if `crdPattern` is empty.
+
+Example: `resources.azure.com/*;containerservice.azure.com/*;keyvault.azure.com/*;managedidentity.azure.com/*;eventhub.azure.com/*`
+
+> **Note:** If you are specifying the Helm values from a shell with `--set crdPattern=...` you must wrap the crdPattern
+> argument in single quotes `'` if it contains wildcard characters the shell would expand. For example:
+> `--set crdPattern='resources.azure.com/*;containerservice.azure.com/*'`
+
+> **⚠️ Warning:** A `crdPattern` of `*` can be used to install all resources, but is not recommended in all cases. 
+> See [best practices](#best-practices) for more details.
```

Comment:
I think maybe simplify this:

```suggestion
> **⚠️ Warning:** A `crdPattern` of `*` may be used to install all resources, but is not recommended. 
> See [best practices](#best-practices) for more details.
```

Reply:
I think the "in all cases" bit weakens the statement and makes it easier to ignore.

Reply:
Fixed here and below.